recipes/digital_rf/meta.yaml

```diff
@@ -0,0 +1,85 @@
+{% set version = "2.6.3" %}
+
+package:
+  name: digital_rf
+  version: {{ version }}
+
+source:
+  url: https://github.com/MITHaystack/digital_rf/releases/download/{{ version }}/digital_rf-{{ version }}.tar.gz
+  sha256: 81624b4cffe727e0ef47369dd2b9b5993b3bfc0f3ce74f34c3f9dcea3a3650d6
+
+build:
+  number: 0
+  # there are dependency conflicts for Python 2 on Windows
+  skip: True  # [win and py2k]
+  run_exports:
+    - {{ pin_subpackage('digital_rf') }}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - cmake >=3.15
+    - make  # [not win]
+
+  host:
+    - hdf5
+    - mako
+    - numpy
+    - pkgconfig
+    - python
+    - setuptools
+
+  run:
+    - h5py
+    - hdf5
+    - {{ pin_compatible('numpy') }}
+    - packaging
+    - python
+    - python-dateutil
+    - pytz
+    - six
+    - watchdog
+
+test:
+  requires:
+    - gnuradio-core
+    - pytest >=3.6
+  imports:
+    - digital_rf
+    - digital_rf.digital_rf_hdf5
+    - gr_digital_rf
+  source_files:
+    - python/tests/*.py
+    - python/examples/*.py
+  commands:
+    - pytest
+    - python python/examples/example_write_digital_metadata.py
+    - python python/examples/example_read_digital_metadata.py
+
+about:
+  home: https://github.com/MITHaystack/digital_rf
+  license: BSD
```

Comment:
Is this BSD-3-Clause?

Reply:
Yes, and that should be a more specific identifier I guess?

Reply:
Yep, exactly :wink: